daemon/src/taker.rs

```diff
@@ -179,10 +179,21 @@ async fn main() -> Result<()> {
     let maker_identity = Identity::new(opts.maker_id);
 
     let seed = Seed::initialize(&data_dir.join("taker_seed")).await?;
+    let (_, identity_sk) = seed.derive_identity();
+
+    // If the wallet seed does not exist, create one using the seed
+    if !&data_dir.join("taker_wallet_seed").exists() {
+        tokio::fs::copy(
+            &data_dir.join("taker_seed"),
+            &data_dir.join("taker_wallet_seed"),
+        )
+        .await?;
+    }
+
+    let wallet_seed = Seed::initialize(&data_dir.join("taker_wallet_seed")).await?;
```

Comment:
Why are we round-tripping through `Seed`? I think we should store the seed words on disk here!

Reply:
I am not sure if there is a huge advantage to doing this. 

We will have to change the implementation of `Seed` or make a seperate `WalletSeed` type which is almost exactly the same as `Seed`. If we change the implementation of `Seed`, we will have to write some migration code. Can also be done in a separate PR but I am not seeing the benefit.

Reply:
The plain seed words are easier to back up and I would say there is some advantage in being able to do that without having to go through the UI.

The migration code is trivial:

- Make `Mnemonic` from existing seed such that getting key from `Mnemonic` is the same key (btw, we need a test that given an existing `Seed` and making a `Mnemonic` from it doesn't change the generated `ExtendedPrivKey`!)
- Write words to disk

What I am getting at here is that in the end, we don't need `Seed` at all for the wallet. On startup, we should:

- Check if seed words exist, if yes, read them and generate `Mnemonic`
- Pass `Mnemonic` to `wallet::Actor`

Reply:
On second thought I think it is worth creating a seperate `WalletSeed`. The current `Seed` is not bip39 compliant.